docs/configuring-playbook-bridge-zulip.md

```diff
@@ -0,0 +1,78 @@
+<!--
+SPDX-FileCopyrightText: 2021 - 2024 Slavi Pantaleev
+SPDX-FileCopyrightText: 2021 Toni Spets
+SPDX-FileCopyrightText: 2024 - 2025 Suguru Hirahara
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+# Setting up a Zulip bridge (optional)
+
+The playbook can install and configure [MatrixZulipBridge](https://github.com/GearKite/MatrixZulipBridge) for you.
+
+See the project's [documentation](https://github.com/GearKite/MatrixZulipBridge/blob/main/README.md) to learn what it does and why it might be useful to you.
+
+## Adjusting DNS records (optional)
+
+By default, this playbook installs the Zulip bridge on the `matrix.` subdomain, at the `/zulip` path (https://matrix.example.com/zulip). This makes it easy to install it, because it **doesn't require additional DNS records to be set up**. If that's okay, you can skip this section.
+
+If you wish to adjust it, see the section [below](#adjusting-the-zulip-bridge-url-optional) for details about DNS configuration.
+
+## Adjusting the playbook configuration
+
+To enable the Zulip bridge, add the following configuration to your `inventory/host_vars/matrix.example.com/vars.yml` file:
+
+```yaml
+matrix_zulip_bridge_enabled: true
+
+# Setting the owner is optional as the first local user to DM `@matrixzulipbridge:example.com` will be made the owner.
+# If you are not using a local user you must set it as otherwise you can't DM it at all.
+matrix_zulip_bridge_owner: "@alice:{{ matrix_domain }}"
```

Comment:
Please add a note about `matrix_admin` var that is set here by default

Reply:
Addressed with https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/4695/commits/8b6d2a31f1452952e6490d2483c4d68ce8e6cb11.